Makefile

```diff
@@ -31,4 +34,11 @@ clean:
 	rm -rf .mypy_cache
 	rm -rf .pytest_cache
 
-.PHONY: install install-dev quality style test test-unit test-integration test-e2e test-smoke test-sanity test-regression build clean
+
+fix:
+	python -m black src tests
+	python -m isort src tests
+	python -m ruff format src tests
+
+
+.PHONY: run install install-dev quality style test test-unit test-integration test-e2e test-smoke test-sanity test-regression build clean fix
```

Comment:
I missed this originally, but we need to update the .PHONY here for what's contained in the file. It's leftover from another repo and contains unsupported functions